app/tests/unit/forms/form/controller.spec.js

```diff
@@ -50,6 +52,21 @@ describe('form controller', () => {
     await controller.readFieldsForCSVExport(req, {}, jest.fn());
     expect(exportService.fieldsForCSVExport).toHaveBeenCalledTimes(1);
   });
+
+  it('should not continue with export if there are no submissions', async () => {
```

Comment:
It's great to see new tests for the backend! It looks like the frontend changes don't have any test coverage, can those be added too? I assume we're still aiming for unit test coverage and the Cypress tests are focused on e2e?